tests/utils/test_graph_utils.py

```diff
@@ -229,6 +230,35 @@ def test_create_adjacency_list_empty(self) -> None:
         assert len(adj_list.keys()) == 0
         assert isinstance(adj_list, defaultdict)
 
+    def test_draw_graph_topology(self) -> None:
```

Comment:
This won't automatically test anything, right?

I suggest we mock the call to networkx, and verify the args passed to the mocked method..

Reply:
So this unit test is a "smokescreen" test IMHO. It makes sure that the code can run without crashing. It is a weak test, but I've inspected the plots and they look good.